### ISSUE TITLE
Pressurization Fan for Stairwells (Zone)

### ISSUE THREAD

**Issue by jeannieboef** [NREL/EnergyPlus#6723]
Issue overview
--------------
Stairwell Pressurization Modelling with AFN

I hope you are well. I'm trying to find a way to insert a pressurization object into the airflownetwork. Having an AirLoop will not solve this, and the simple ZoneVentilation object don't do it either as they don't work with AFN. Is their a supply pressurisation fan equivalent to the ExhaustFan object, or another way to do this?

Dr. GU says no, but it would be an easy addition.

**Comment by mjwitte** [comment]
@jeannieboef As of v8.8 and beyond, an AirLoop can be pressurized (less return than supply, or even no return at all). I don't know if this feature is compatible with AFN, but it's worth looking at modeling it this way.

**Comment by jeannieboef** [comment]
Hi Mike,

Are you refering to the major revamp of the airloop as required for
multiple loops per zone? V8.8 has just come out,  right?

Based in my v8.7
I tried this but you'd have to model the AFN 'with distribution' which is
really focused on leaks in the duct work.

Futhermore, airloops require an exhaust.  There exists no supply only fan,
though the zoneexhaust equivalent does exist.

The only way I know of to reduce the airloop exhaust air mass is to add
other extraction via zoneexhaust fans or ZoneMixing objects. I don't think
either of those will pressurise the zone. Simple zone ventilation objects
like ZoneMixing or ZoneVentilation are turned off during AFN simulations.
....
So you reckon this is doable with v8.8? I'll try it out.

Thanks!

**Comment by jeannieboef** [comment]
On second thought I don't think this would work. In the past you could link
a zoneexhaust fan into the AFN, but the link was not automatic. Care had to
be taken to use the exact name of the zone equipment in certain AFN
objects.  But it was not possible to link the terminal units of an airloop
(except when modelling "with distribution", as far as I understand... Which
would be a lot of overhead that's not required).

**Comment by mjwitte** [comment]
True, pressurizing with an air loop would require adding distribution to the AFN. I will tag this as a new feature request.  Just for completeness here, in v8.7 there is also the "Zone Return Air Flow Rate Fraction Schedule Name" field in ZoneHVAC:EquipmentConnections which can be used to pressurize a zone.

**Comment by jasondegraw** [comment]
This would definitely be a new feature. CONTAM-style stair pressurization isn't possible with the current capabilities of AFN.
